codelists/partyRole.csv

```diff
@@ -0,0 +1,6 @@
+Code,Title,Description
```

Comment:
The filename should be `+partyRole.csv`.

Reply:
With a leading plus sign? Why?

Reply:
Because we're "adding" codes to the existing file. Without the "+", the file gets replaced. (We can similarly remove codes by prefixing with "-".)